doc/source/overview/getting_started.md

```diff
@@ -37,12 +37,52 @@ pip install alibi[shap]
 :class-label: sd-pt-0
 :sync: label-dist
 ```{div} sd-mb-1
-Installation with support for distributed computation of explanations.
+Installation with support for AnchorTabularDistributed and 
```

Comment:
I would remove `AnchorTabularDistributed` from the docs as it's not really well tested to understand potential improvements. I would consider it private (even though it's accessible by default).